doc/nrf/ug_thingy91.rst

```diff
@@ -171,50 +171,28 @@ Building and programming using VS Code extension
 Complete the following steps to build |NCS| projects with |VSC|:
 
 1. Start |VSC|.
-
    If you installed the |NCS| using the :ref:`gs_app_tcm`, you can click the :guilabel:`Open VS Code` button next to the version you installed.
-
-#. Click :guilabel:`Create new application from sample` from the :guilabel:`Welcome` panel.
-
+#. In the Welcome panel or on the Welcome page, click :guilabel:`Add an existing application`.
    The :guilabel:`New Application` screen opens.
-
-#. Provide the following information to create the application:
-
-   a. Select the :guilabel:`Freestanding` radio button for the :guilabel:`Application type`.
-      This links the application to the version of |NCS| you have installed.
-   #. Select the |NCS| version from the :guilabel:`nRF Connect SDK` drop-down menu and the corresponding toolchain version from the :guilabel:`nRF Connect Toolchain` drop-down menu.
-      The version you selected when setting up |VSC| is set by default.
-   #. Choose the location where the application is saved.
-      To do this, either type the path in the :guilabel:`Application location` field, or browse to the desired location by clicking the :guilabel:`...` button.
-   #. Select the application template that you want to use.
-
-      You can either select one from the :guilabel:`Application template` drop-down menu, or browse the available templates with a preview of their documentation in the :guilabel:`Browse Samples` screen by clicking the :guilabel:`Select...` button.
-      Both the drop-down and the :guilabel:`Browse Samples` screen contain the same applications.
-   #. Enter the :guilabel:`Application name`.
-
-      When you create the application, this creates a new folder in the :guilabel:`Application folder` directory with the same name.
-
+#. Navigate to the directory of the nRF9160-compatible sample or application you want to work with.
+   Samples are located under :file:`nrf/samples/`.
+   Applications are located under :file:`nrf/applications/`.
```

Comment:
Since line breaks are not kept in the build, I'd use something like this instead:
```suggestion
#. Navigate to the directory of the nRF9160-compatible sample or application you want to work with.

   Samples are located under :file:`nrf/samples/`, and applications under :file:`nrf/applications/`.
```
This way is clearer what is part of the step itself, and what is additional information for doing the step.